<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confBD</class>
 <widget class="QWidget" name="confBD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>244</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>282</width>
    <height>244</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>282</width>
    <height>244</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Conexión con la BD</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="2">
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="text">
      <string>Cyriax</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Driver</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Host Name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Database Name</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>115</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ACEPTAR</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>MYSQL</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>115</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CANCELAR</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PARÁMETROS DE LA BASE DE DATOS</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Cantarell</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
